boot/freeldr/freeldr/include/fs/stat.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     Dual-licensed:
+ *              LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ *              MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     sys/stat.h S_IF* flags used for stat/inode.st_mode, and S_IS*
+ *              convenience macros, used by *nix-based file-systems (EXTn, Btrfs, ...)
+ * COPYRIGHT:   Copyright 2025 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
+ *
+ * NOTE: Based on definitions from MinGW-w64, (u)CRT, schily/stat.h, etc.
+ */
+
+#ifndef _INC_STAT_FLAGS
+#define _INC_STAT_FLAGS
+
+#pragma once
```

Comment:
Is guard ifdef needed even with `#pragma once`?

Reply:
No i'll remove the guard, since also it doesn't define an "official"/documented name that other headers check.